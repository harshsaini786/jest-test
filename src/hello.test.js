const hello = require("./hello");

test("greets the user", () => {
  expect(hello("John")).toBe("Hello John");
});
