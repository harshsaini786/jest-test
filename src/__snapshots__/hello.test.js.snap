// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`greets the user 1`] = `"Hello John"`;
